src/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -362,6 +378,51 @@ public static ThumbnailGenerationTask getBitmapWorkerTask(ImageView imageView) {
         return null;
     }
 
+    public static Bitmap addVideoOverlay(Bitmap thumbnail){
+        Bitmap playButton = BitmapFactory.decodeResource(MainApp.getAppContext().getResources(),
+                R.drawable.view_play);
+
+        Bitmap resizedPlayButton = Bitmap.createScaledBitmap(playButton,
```

Comment:
@AndyScherzinger can I crop the images?
Scaling (or at least resizing) is still necessary as the visual center of the play button is not the center of the image.